packages/twenty-server/src/engine/metadata-modules/remote-server/remote-server.service.ts

```diff
@@ -99,6 +104,98 @@ export class RemoteServerService<T extends RemoteServerType> {
     );
   }
 
+  async updateOneRemoteServer(
+    remoteServerInput: UpdateRemoteServerInput<T>,
+    workspaceId: string,
+  ): Promise<RemoteServerEntity<RemoteServerType>> {
+    validateObject(remoteServerInput.foreignDataWrapperOptions);
```

Comment:
I struggled to understand what validateObject does as it's very broad, I think we should fine a more explicit name @thomtrp